### DISCUSSION TITLE
Latest branch for Vidydaan

### DISCUSSION THREAD

**Discussion by agamdahiya** [Sunbird-coKreat/Community#55] [Q&A]
i wanted to know which is the latest branch for sunbird RC - opensaber (https://github.com/Sunbird-RC/open-saber/branches)
for the vidyadaan . Because in this there is release 2.0.3 which  is the latest release but there is a branch also named as Vidyadaan in the branches . And these two branches the latest release and the vidydaan are almost 2 years old branches.

So which branch should i clone for the Vidyadaan Portal and is there any environment variables for the branch ?

**Comment by arun1205** [comment]
@agamdahiya u can clone https://github.com/Sunbird-RC/open-saber/tree/vidyadaan 
You can visit https://github.com/Sunbird-RC/open-saber/wiki for further information
